runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillStateReader.java

```diff
@@ -195,6 +196,7 @@ public ValuesAndContPosition(List<T> values, @Nullable ContinuationT continuatio
   private final ByteString key;
   private final long shardingKey;
   private final long workToken;
+  private final Supplier<AutoCloseable> readWrapperSupplier;
```

Comment:
Add comment, something like
// Blocking reads should be wrapped with a closable vended by readWrapperSupplier

Reply:
Done